x/gamm/keeper/genesis_test.go

```diff
@@ -111,7 +109,6 @@ func TestGammExportGenesis(t *testing.T) {
 
 	genesis := app.GAMMKeeper.ExportGenesis(ctx)
 	require.Len(t, genesis.Pools, 2)
-	require.Equal(t, genesis.PoolCount, uint64(2))
```

Comment:
shouldn't we be checking NextPoolID here?

Reply:
Next pool id is tracked in the swap router now. All increment logic was removed from gamm. The reason we still keep the index is for migrations